<?xml version="1.0"?>

<launch>
	<node pkg="optitrack_listener" name="simu_optitrack_listener_new" type="simu_optitrack_listener_new" output="screen">
		<param name="model_name" type="string" value="iris" />
		<param name="output_topic_name" type="string" value="/drone/odom"/>
		<param name="output_ref_frame" type="string" value="enu" />
		<param name="publish_odom_pose" value="true" />
	</node>
</launch>